doc/build-unix.md

```diff
@@ -61,34 +61,40 @@ Dependency Build Instructions: Ubuntu & Debian
 ----------------------------------------------
 Build requirements:
 
-	sudo apt-get install build-essential libtool autotools-dev autoconf pkg-config libssl-dev libevent-dev
+        sudo apt-get install build-essential libtool autotools-dev autoconf pkg-config libssl-dev libevent-dev
```

Comment:
@MarcoFalke: right.

So either:
[4 spaces][code]

Or:
\`\`\`
[code]
\`\`\`

@laanwj: content itself looks good to me!